package.json

```diff
@@ -36,7 +36,6 @@
     "webpack-stream": "3.1.0"
   },
   "devDependencies": {
-    "browser-sync": "2.11.2",
```

Comment:
`devDependencies` don't get installed on Travis, while `dependencies` do. Since `browser-sync` is only used locally, it would be preferable in `devDependencies`.